### PR TITLE
Add helm values for settings.spec.ratelimit server

### PR DESCRIPTION
**Pull request by andrzej-talarek**
# Description

ratelimitServer element is missing in Helm Chart.

# Context

RL can't be enabled, when deploying Gloo Edge using Helm Chart.
Cherry pick of https://github.com/solo-io/gloo/pull/8190

# Checklist:

[*] If I updated APIs (our protos) or helm values, I ran make -B install-go-tools generated-code to ensure there will be no code diff
[*] I followed guidelines laid out in the Gloo Edge [contribution guide](https://docs.solo.io/gloo-edge/latest/contributing/)
[*] I opened a draft PR or added the work in progress label if my PR is not ready for review
[*] I have performed a self-review of my own code
[*] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: install/helm/gloo/generate/values.go [hunk 1]
```diff
@@ -203,6 +203,7 @@ type Settings struct {
...
 	Aws                           AwsSettings             `json:"aws,omitempty"`
 	RateLimit                     interface{}             `json:"rateLimit,omitempty" desc:"Partial config for Gloo Edge Enterprise’s rate-limiting service, based on Envoy’s rate-limit service; supports Envoy’s rate-limit service API. (reference here: https://github.com/lyft/ratelimit#configuration) Configure rate-limit descriptors here, which define the limits for requests based on their descriptors. Configure rate-limits (composed of actions, which define how request characteristics get translated into descriptors) on the VirtualHost or its routes."`
+    RatelimitServer               interface{}             `json:"ratelimitServer,omitempty" desc:"External Ratelimit Server configuration for Gloo Edge Open Sources’s rate-limiting service, based on Envoy’s rate-limit service; supports Envoy’s rate-limit service API. (reference here: https://docs.solo.io/gloo-edge/master/guides/security/rate_limiting/)"`
```
[the remark this answers is not in the document]
Comment (andrzej-talarek) [reply install/helm/gloo/generate/values.go]: yeap. I don't code go - didn't know the conventions and indents looked the same in intellij :D.

**Comment by andrzej-talarek** [comment]
Sorry, there was a conflict in `values.go` file and I've resolved it manually - putting 4 spaces instead of a tab as padding solution.
Fixed that - no more mismatch there.

**Comment by andrzej-talarek** [comment]
`runtime error: invalid memory address or nil pointer dereference` looks like some random problem - can you please reschedule tests?

**Review by jbohanon — APPROVED** [review]

**Pull request merged**